### PR TITLE
docs(relnote): Update Fx115 release notes

### PR DESCRIPTION
**Pull request by dipikabh**
This PR updates release notes for Fx115 (released on July 04) and preps release note pages for Fx116 and Fx117.

### CODE REVIEW

**Comment by bsmth** [comment]
Taking a look 👀

File: files/en-us/mozilla/firefox/releases/115/index.md
Comment (bsmth) [inline files/en-us/mozilla/firefox/releases/115/index.md:1]: Should we get rid of the empty sections (`### Developer Tools`, `#### Removals`) while we're at it?

Comment (dipikabh) [reply files/en-us/mozilla/firefox/releases/115/index.md:1]: Yes! Done in https://github.com/mdn/content/pull/27885/commits/776ddeee7010e86d0f164f7f92b51047195800b7.

**Review by bsmth — APPROVED** [review]
Thanks, Dipika 👍🏻

**Pull request merged**